lib/src/cmaf/ham/mapper/hls/mapHamToHls.ts

```diff
@@ -51,7 +51,16 @@ function mapHamToHls(presentations: Presentation[]): Manifest {
 					const { mainRef, playlist } =
 						generateManifestPlaylistPiece(track);
 					mainManifest += mainRef;
-					playlists.push({ manifest: playlist, type: 'hls' });
+					const manifestFileName =
+						track.fileName?.includes('http') ||
```

Comment:
Would be a problem if we have chunks locally?

Reply:
I put this if statement here because if fileName is something like : `https://www.path-to-file.com/fileName.m3u8` then the file will be written using that name but that doesn't work because the name of a file must not include : or / characters. 

I tested all the samples that we have and all of them worked well. Having chunks locally should not be a problem.